### ISSUE TITLE
Planned unscheduled departure?

### ISSUE THREAD

**Issue by christoffersoltau** [Kemmey/Kemmey-TeslaWatch-Public#285]
Tesla allows us to set a fixed schedule where the car can be automatically prepared for departure at a specific time. 

Sometimes - and especially in these working from home weird times - we leave home at all kinds of weird hours... and often WE forget to preheat the car during winter, which is totally MY mistake :)

could the app handle setting a short term / one term “turn on climate tomorrow at xx o’clock” so that if the schedule says 7 every morning, then specifically tomorrow it would be ready for departure at 5 ( as an example obviously). That way it could be set the evening before while remembered, instead of 5 mins before departure :)

**Comment by Kemmey** [comment]
YES! But the absolute easiest way to do that will be when I ship proper Siri/shortcuts support, because then you would just make a shortcuts automation with a date/time trigger. So not something I'm looking to build into the app - but that definitely will be possible in future via shortcuts. 

If you do want that particular feature today, look no further than Stats app :-D

**Comment by Kemmey** [comment]
@christoffersoltau 

I think you'll like the next beta :-D 

![image](https://user-images.githubusercontent.com/4688084/107994069-2ccc3200-6fdc-11eb-8fc8-fdc679010f68.jpeg)

![image](https://user-images.githubusercontent.com/4688084/107994092-30f84f80-6fdc-11eb-80be-c5cd23698048.png)

![image](https://user-images.githubusercontent.com/4688084/107994096-35246d00-6fdc-11eb-930e-7662c2106e3c.png)
